Plugins/Flow.Launcher.Plugin.Program/Programs/Win32.cs

```diff
@@ -381,6 +381,7 @@ private static string Extension(string path)
         private static IEnumerable<Win32> UnregisteredPrograms(List<ProgramSource> sources, string[] suffixes, string[] protocols)
         {
             var paths = ExceptDisabledSource(sources.Where(s => Directory.Exists(s.Location) && s.Enabled)
```

Comment:
Have you tested whether parallel give performance boost? In my very early test it is not so significant somehow

Reply:
On my computer it is about 30% faster. I will do some more tests and show you the results.

Reply:
Parallel gives up to 30% boost on my computer. Not always a significant boost but generally faster.